lerna.json

```diff
@@ -1,6 +1,11 @@
 {
-  "packages": [
-    "packages/*"
-  ],
-  "version": "1.0.0"
+  "packages": ["packages/*"],
+  "version": "independent",
+  "changelogPreset": "angular",
+  "command": {
+    "version": {
+      "message": "chore(release): publish %s",
```

Comment:
just looking at [the commits in this PR](https://github.com/mitchelvanbever/babel-vite/pull/1/commits/13ac1d7ee5de4cfdbd6499010c07be0d138fbde7), I don't think the `%s` is working.

Reply:
Yeah correct, this only works with fixed versioning. I would suggest to make it `chore(release): published new version(s)`. The multiline commit already includes the new versions.